src/modules/project/pages/projectCreate/views/ProjectCreateStart.tsx

```diff
@@ -74,8 +74,10 @@ export const ProjectCreateStart = () => {
 
         {!loading && (!isLoggedIn || !userHasProjectCreatableAccounts) ? (
           <VStack w="100%">
-            <Text color="neutral.700" pb={3}>
-              {t('You need to login before creating your project.')}
+            <Text align="center" color="neutral.700" pb={3}>
```

Comment:
This copy is good for users that are already logged in. Otherwise, we'd keep the current copy. 

Basically, you'd want to split the logic on line 75: `!isLoggedIn || !userHasProjectCreatableAccounts`